internal/common/common.go

```diff
@@ -393,6 +393,16 @@ func GetTagFromImageRef(ref string) string {
 	}
 }
 
+func GetAPIVipDNSName(c *Cluster) string {
```

Comment:
Consider renaming to GetIgnitionEndpoint?

Reply:
I disagree, the literal APIVipDNSName cluster field gets set with the value being the string returned by this function